src/settings_csv.py

```diff
@@ -22,6 +23,7 @@
 TICKER_HUAHUA = "HUAHUA"
 TICKER_IOTEX = "IOTX"
 TICKER_JUNO = "JUNO"
+TICKER_STARGAZE = "STARGAZE"
```

Comment:
"STARS" instead?  I'm not very familiar with Stargaze but my understanding STARS is the ticker?  Using ticker just helps keep it consistent.

Reply:
fixed! 🤦  i'll do some more testing myself as well tomorrow